crates/rpc/rpc-types-compat/src/block.rs

```diff
@@ -61,30 +64,16 @@ pub fn from_block_full(
     block_hash: Option<B256>,
 ) -> Result<Block<WithOtherFields<Transaction>>, BlockError> {
     let block_hash = block_hash.unwrap_or_else(|| block.block.header.hash_slow());
-    let block_number = block.block.number;
-    let base_fee_per_gas = block.block.base_fee_per_gas;
-
     // NOTE: we can safely remove the body here because not needed to finalize the `Block` in
     // `from_block_with_transactions`, however we need to compute the length before
     let block_length = block.block.length();
-    let body = std::mem::take(&mut block.block.body);
-    let transactions_with_senders = body.into_iter().zip(block.senders);
-    let transactions = transactions_with_senders
-        .enumerate()
-        .map(|(idx, (tx, sender))| {
-            let tx_hash = tx.hash();
-            let signed_tx_ec_recovered = tx.with_signer(sender);
-            let tx_info = TransactionInfo {
-                hash: Some(tx_hash),
-                block_hash: Some(block_hash),
-                block_number: Some(block_number),
-                base_fee: base_fee_per_gas.map(u128::from),
-                index: Some(idx as u64),
-            };
+    let body = mem::take(&mut block.block.body);
+    let senders = block.senders;
```

Comment:
please revert this change, it is unrelated

Reply:
yes, unclear how this is relevant here, please submit separately

Reply:
thanks. ended up using <https://github.com/paradigmxyz/reth/pull/10600> changes here instead, will revert.